src/app/services/globalSearch/globalSearch.service.ts

```diff
@@ -0,0 +1,12 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class GlobalSearchService {
+  constructor(private readonly http: HttpClient) {}
+
+  public getGlobalSearchServices(): Observable<GlobalSearchService[]> {
```

Comment:
Hi @Daniel-Lam, this might be because this branch was created well before, and this is a new file, did not come up as a conflict when merging global search base branch. I have now deleted the file.